mmdet/models/losses/gfocal_loss.py

```diff
@@ -77,6 +77,18 @@ def distribution_focal_loss(pred, label):
     return loss
 
 
+@mmcv.jit(derivate=True, coderize=True)
+@weighted_loss
+def ld_loss(pred, label, soft_label, T):
+    ld_loss = F.kl_div(
+        F.log_softmax(pred / T, dim=1),
+        F.softmax(soft_label / T, dim=1).detach(),
```

Comment:
better make the `.detach` as an option in the input.

Reply:
You mean support online distillation?